inc/class-plugin-dependency.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace TMSC;
+
+class Plugin_Dependency {
+
+	/** @type string Basename for the plugin with the dependency  **/
```

Comment:
Not a blocker, but this does not follow php docblock standards - https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#2-1-class-members